test/spec/unit/adapters/analytics/AnalyticsAdapter_spec.js

```diff
@@ -59,92 +57,124 @@ FEATURE: Analytics Adapters API
+          events.emit(eventType, args);
 
-        it(`THEN should call \`window.${config.global}\` function\n`, () => {
-          assert.ok(spyTestGlobal.args[4][1] === eventType, `with expected event type\n`);
-          assert.deepEqual(spyTestGlobal.args[4][2], args, `with expected event data\n`);
+          assert.ok(spyTestGlobal.args[0][1] === eventType, `with expected event type\n`);
+          assert.deepEqual(spyTestGlobal.args[0][2], args, `with expected event data\n`);
         });
-        window[config.global].restore();
-      });
 
-      describe(`WHEN a bidTimeout event occurs\n`, () => {
-        const eventType = BID_TIMEOUT;
-        const args = { call: 'timeout' };
-        const adapter = new AnalyticsAdapter(config);
-        var spyTestGlobal = sinon.spy(window, config.global);
+        it('SHOULD call global when a bidResponse event occurs', () => {
+          const eventType = BID_RESPONSE;
+          const args = { call: 'response' };
 
-        adaptermanager.registerAnalyticsAdapter(adapter, 'adapter');
-        adapter.enableAnalytics();
-        events.emit(eventType, args);
+          adapter.enableAnalytics();
+          events.emit(eventType, args);
 
-        it(`THEN should call \`window.${config.global}\` function\n`, () => {
-          assert.ok(spyTestGlobal.args[5][1] === eventType, `with expected event type\n`);
-          assert.deepEqual(spyTestGlobal.args[5][2], args, `with expected event data\n`);
+          assert.ok(spyTestGlobal.args[0][1] === eventType, `with expected event type\n`);
+          assert.deepEqual(spyTestGlobal.args[0][2], args, `with expected event data\n`);
         });
-        window[config.global].restore();
-      });
 
-      describe(`WHEN adapter.enableAnalytics is called twice\n`, () => {
-        const eventType = BID_TIMEOUT;
-        const args = { call: 'timeout' };
-        const adapter = new AnalyticsAdapter(config);
-        var spyTestGlobal = sinon.spy(window, config.global);
+        it('SHOULD call global when a bidTimeout event occurs', () => {
+          const eventType = BID_TIMEOUT;
+          const args = { call: 'timeout' };
 
-        adaptermanager.registerAnalyticsAdapter(adapter, 'adapter');
-        adapter.enableAnalytics();
-        events.emit(eventType, args);
+          adapter.enableAnalytics();
+          events.emit(eventType, args);
 
-        it(`THEN should call \`window.${config.global}\` function\n`, () => {
-          assert.ok(spyTestGlobal.args[5][1] === eventType, `with expected event type\n`);
-          assert.deepEqual(spyTestGlobal.args[5][2], args, `with expected event data\n`);
+          assert.ok(spyTestGlobal.args[0][1] === eventType, `with expected event type\n`);
+          assert.deepEqual(spyTestGlobal.args[0][2], args, `with expected event data\n`);
         });
-        window[config.global].restore();
+
+        it('SHOULD NOT call global again when adapter.enableAnalytics is called with previous timeout', () => {
```

Comment:
This test originally said "WHEN adapter.enableAnalytics is called twice" and was testing that the event was present from the test that executed before it.  I updated it to test what I think it was trying to test, but let me know if I'm mistaken.